index.js

```diff
@@ -1,11 +1,50 @@
 'use strict';
 
 /**
- * Adds commas to a number
- * @param {number} number
- * @param {string} locale
+ * Push and pop from a stack
+ * @param {string} value
+ * @param {number} value
  * @return {string}
+ * @return {array}
  */
-module.exports = function(number, locale) {
-    return number.toLocaleString(locale);
-};
+
+//  creating a stack of length 5
+let lengthOfStack = 5
+let stack = new Array(lengthOfStack);
+let lastUpdatedIndex = 0;
+
+let addValueToStack = (value) => {
+// pushing value onto the stack
```

Comment:
lets add function level  comments with input output

Reply:
can you please elaborate

Reply:
basically same from above, add individual comment for function